apps/builder/app/canvas/canvas.tsx

```diff
@@ -145,7 +145,10 @@ const DesignMode = () => {
   useEffect(subscribeInstanceSelection, []);
   useEffect(subscribeInstanceHovering, []);
   useEffect(subscribeInspectorEdits, []);
-  useMount(subscribeFondLoadingDone({ signal }));
+  useMount(subscribeFontLoadingDone({ signal: abortController.signal }));
+  useEffect(() => () => {
+    abortController.abort();
```

Comment:
Ok, so you pass one abort c0ntroller signal and abort another? See, you recreate it on every render. You can unsubcribe in useMount callback. Do not expose this just to use abort controller.

Reply:
We use subscribe(): unsubscribe pattern everywhere, why change?

Reply:
This way you can pass the same signal to all and reuse the abort controller. This is a new pattern we discussed with Ivan, since we learned about abort controller. Seems like a much better option than just returning a callback. If you want to discuss this, lets do in dm

Reply:
Oh you are right about where I create the controller, it should be only created once, fixing it